tests/run/named-tuples.check

```diff
@@ -0,0 +1,9 @@
+(Bob,33)
```

Comment:
Not with `.toString`, since the representation of named and unnamed tuples is exactly the same. 

But we really should have a standard `Show` typeclass that could handle these things.